jsFiles/utility.js

```diff
@@ -1,71 +1,89 @@
-// displayWeatherUserLocation() is used to get the location of the user's computer: latitude and longitude
-//(geographical coordinates). These data is used to build the url for Open Weather API,
-// where the data of the weather data is fetch from.
+/* displayWeatherUserLocation() is used to get the location of the user's computer: latitude and longitude
```

Comment:
don't forget about indentation. The format is standard so it always should be that way:
```js
/**
 * Description starts here
 * 
 * @param {type} your ardument description
 */
```

Reply:
I got it now, sorry, now @param turn blue, I think it is a good sign and the app stills works :)